Source/Engine/Input/Input.h

```diff
@@ -293,7 +293,13 @@ API_CLASS(Static) class FLAXENGINE_API Input
     /// Event fired when virtual input action is triggered. Called before scripts update. See <see cref="ActionMappings"/> to edit configuration.
     /// </summary>
     /// <seealso cref="InputEvent"/>
-    API_EVENT() static Delegate<StringView> ActionTriggered;
+    API_EVENT() static Delegate<StringView, const InputActionState&> ActionTriggered;
```

Comment:
This can be just `InputActionState` as it's enum value

Reply:
Like instead of `const InputActionStat&`?

Reply:
Yea, we would pass just value of enum instead of const reference. Not a big deal, just a minor suggestion for readability fo this event in C++.